release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -95,6 +95,51 @@ You can now install a cluster on Google Cloud Platform (GCP) and use a personal
 
 //Link for more info when available.
 
+[id="ocp-4-7-osp-bare-metal-machines"]
+==== Installing a cluster on {rh-openstack} that uses bare metal machines
+
+You can now install a cluster on {rh-openstack-first} that uses bare metal machines. The cluster can have both control plane and compute machines running on bare metal, or just compute machines.
+
+// Link when available.
+
+[id="ocp-4-7-osp-validations"]
+==== Validating {rh-openstack} requirements at installation
+
+The {product-title} installer now performs a variety of checks before attempting to install a cluster on {rh-openstack}. The installer can identify certain conditions that cause installations to fail, such as overlapping subnets.
+
+[id="ocp-4-7-osp-custom-subnets"]
+==== Custom subnets for machines on {rh-openstack}
+
+You can now configure machines in clusters that run on {rh-openstack} to use a network and subnet of your choice.
+
+// Link when available. Custom subnets, again. 
+
+[id="ocp-4-7-upi-playbooks"]
+==== Easier access to {rh-openstack} user-provisioned infrastructure playbooks
+
+Ansible playbooks for installing a cluster on your own {rh-openstack} infrastructure are now packaged for retrieval by using a script in the installation documentation.
+
+// Link when available. Custom subnets section. 
+
+[id="ocp-4-7-sr-iov"]
+==== Support for SR-IOV machines on {rh-openstack}
```

Comment:
It might be worth saying that this is not supported with Kuryr (probably works, but there are QE concerns). Also I believe only secondary SR-IOV interfaces are supported.

Reply:
Thanks. The docs for this are still in dev review, so I don't have a complete picture, here.

Reply:
This item is still up in the air. I'll confirm whether or not to conclude it immediately prior to publication.